application/templates/index.html

```diff
@@ -0,0 +1,341 @@
+						<p>Sprints</p>
+					</td>
+					<td class="ce19">
+						<p>0:40 up res. 0:50 up res. 1:10-1:25(15s) Sprint.</p>
+						<p>1:55up res 2:10 up res. 2:25-2:40(15s) Sprint. </p>
+						<p>2:55 start building up. 3:14-3:34(20s)</p>
+					</td>
+				</tr>
+				<tr class="ro6">
+					<td class="ce4">
+						<p>6</p>
+					</td>
+					<td class="ce9">
+						<p>Uprising</p>
+					</td>
+					<td  class="ce12">
+						<p>Muse</p>
+					</td>
+					<td class="ce16">
+						<p>05:05</p>
+					</td>
+					<td  class="ce12">
+						<p>Standing Climb</p>
+					</td>
+					<td class="ce9">
+						<p>0:41. 1.03. 1:18. 1:26 (urn up &amp; stand up) 1.55(Sit down only). 2:19. 2:32. 2:40( up &amp; stand up). </p>
+						<p>3:16 (sit down &amp; take abit of and recover).</p>
+						<p>3:49 (start adding abit). 4:07 (big amount and stand up) hold for the last minute</p>
+					</td>
+				</tr>
+				<tr class="ro2">
+					<td class="ce2">
+						<p>7</p>
+					</td>
+					<td class="ce5">
+						<p>Hot right now</p>
+					</td>
+					<td  class="ce5">
+						<p>DJ Fresh, Rita ora</p>
+					</td>
+					<td class="ce14">
+						<p>03:00</p>
+					</td>
+					<td  class="ce5">
+						<p>speed endurance</p>
+					</td>
+					<td class="ce19">
+						<p>30s to get set. 2:30 endurance.</p>
+					</td>
+				</tr>
+				<tr class="ro7">
+					<td class="ce3">
+						<p>8</p>
+					</td>
+					<td class="ce7">
+						<p>Love yourself</p>
+					</td>
+					<td  class="ce7">
+						<p>Billy Porter</p>
+					</td>
+					<td class="ce15">
+						<p>04:12</p>
+					</td>
+					<td  class="ce7">
+						<p>standing jog, waves, seated finsih</p>
+					</td>
+					<td class="ce20">
+						<p>start standing jog. 0:38-0:53 back and hold. 0:53- 1:09 1x1 up and down. </p>
+						<p>1:09 up and hold. 1:39-1:54  back and hold. 1:54-2:09  1x1 up and down.  </p>
+						<p>2:09- 2:40 up and hold. 2:40-3:03 back in hold. 3:03-3:33 1x1 to end  </p>
+						<p>Cut track here</p>
+					</td>
+				</tr>
+				<tr class="ro3">
+					<td class="ce2">
+						<p>9</p>
+					</td>
+					<td class="ce5">
+						<p>Take me home - radio edit</p>
+					</td>
+					<td  class="ce5">
+						<p>Cash cash, Bebe Rexha</p>
+					</td>
+					<td class="ce14">
+						<p>03:26</p>
+					</td>
+					<td  class="ce5">
+						<p>Final climb</p>
+					</td>
+					<td class="ce19">
+						<p>up when she says 'home' 7 1 7 (15) hold for 55s - the Last lot we don’t do</p>
+					</td>
+				</tr>
+				<tr class="ro6">
+					<td class="ce3">
+						<p>10</p>
+					</td>
+					<td class="ce10">
+						<p>Lifestyle of the rich and famous</p>
+					</td>
+					<td  class="ce10">
+						<p>Good Charlotte</p>
+					</td>
+					<td class="ce17">
+						<p>03:10</p>
+					</td>
+					<td  class="ce10">
+						<p>sprints</p>
+					</td>
+					<td class="ce21">
+						<p>Start seated. 0:28 up resis stand up jog. 0:50 leave resis sit andsprint(18 sec).</p>
+						<p>1:27 up resis stand up jog. 1:49 leave resis sit and sprint.</p>
```

Comment:
A good first step towards using a db to store your info would be to store this in a python object and then use jinja magic to bring it in.